department-of-veterans-affairs/caseflow
# HunJerBAH APPEALS-23300: IssueUpdate Task Created when MST/PACT Edited

Resolves [APPEALS-23300](https://jira.devops.va.gov/browse/APPEALS-23300?ticket=ST-30287-iy0YXOy6eYqiZv6SKt7XHSFOGm0-login.max.gov)

# Description
Updated logic for Edit page issue creation, edit, and removal, as well as created IssueUpdate task creation for MST/PACT edits. 

## Acceptance Criteria

## Testing Plan
1. Sign in as BVADWISE or another admin intake user
2. Intake a new appeal with MST/PACT issues or find an existing appeal with issues on it.
3. Navigate to the appeal's case detail page. Click the **correct issues** link.
4. Change the MST/PACT status of an issue (either by removing MST/PACT, or adding MST/PACT). 
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/e7170b18-69ed-4f69-8ed5-310551af2df9)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/d6888612-6712-412d-946b-2276608945c3)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/3ece606d-e407-4f55-b89d-d624fdc1fffb)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/08f5eaa5-537d-47bc-8e5f-c14b0f4db973)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/520337ce-67e5-48f7-b7f1-866e7e93d47f)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/d7d9ec79-5195-44df-a304-87f60c795161)

6. click **save**.
7. Navigate to the case timeline and notice the IssueUpdate task that was created. 
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/3c63dfd3-2247-461e-bd19-e98a8f4449da)
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/38e593b5-da30-4eee-aecc-3b49c6bfe784)
